compiler/rustc_typeck/src/astconv/mod.rs

```diff
@@ -1266,7 +1266,7 @@ impl<'o, 'tcx> dyn AstConv<'tcx> + 'o {
         } else {
             err.span_suggestion(
                 span,
-                "use fully-qualified syntax",
+                "use fully qualified syntax",
```

Comment:
Why was the hyphen here removed? It seems correct to me.

Reply:
I agree; I think it's correct with the hyphen. The issue is that it's referred to *without* the hyphen everywhere else in the compiler and in the [book](https://doc.rust-lang.org/book/ch19-03-advanced-traits.html#fully-qualified-syntax-for-disambiguation-calling-methods-with-the-same-name). I figure that unless and until we change it elsewhere, it should be consistent.

Reply:
See `@steveklabnik`'s comment: https://github.com/rust-lang/rust/pull/76138#discussion_r485706393.